### PR TITLE
PR 51722 patch for 16.2 

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**
See https://github.com/angular/angular/pull/51722#issuecomment-1719135313

### CODE REVIEW

File: packages/core/src/signals/src/graph.ts [hunk 1]
```diff
@@ -26,7 +26,11 @@ export function setActiveConsumer(consumer: ReactiveNode|null): ReactiveNode|nul
   return prev;
 }
 
-export const REACTIVE_NODE = {
+export function isInNotificationPhase(): boolean {
```
Comment (JoostK) [inline packages/core/src/signals/src/graph.ts:29]: This can be removed from this change I believe (it was the reason for the conflict)

Comment (pkozlowski-opensource) [reply packages/core/src/signals/src/graph.ts:29]: Oh, you are right. I'm assuming that main / 16.2 discrepancy comes from the effects scheduling change?

Comment (JoostK) [reply packages/core/src/signals/src/graph.ts:29]: Yep, I think that's where the things diverged.

**Review by crisbeto — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 7b40d74fb438805069a72e26b68ea8068c737fd2.

**Pull request closed without merging**